tests/cloud_operations/packer_image_builder/test_plan.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2021 Google LLC
```

Comment:
I know it's a pain but all other tests use 2 spaces for indentation

sorry...

Reply:
No problem, I will change it. BTW I used `tests/networking/filtering_proxy` as an example, you will find 4 spaces there ;-)

Reply:
good catch :) can you also fix that?

Reply:
Sure, fixed.